test/test_linearizer.py

```diff
@@ -191,14 +186,98 @@ def test_early_end_local(self):
     self.assertLess(k.uops.uops.index(barriers[0]), k.uops.uops.index(ifs[0]))
     self.assertLess(k.uops.uops.index(ifs[0]), k.uops.uops.index(endifs[0]))
     self.assertLess(k.uops.uops.index(barriers[1]), k.uops.uops.index(ifs[1]))
-
-  @unittest.expectedFailure
+    x = Tensor.randn(3,27,32).realize()
+    helper_linearizer_ast(ast, [x], wanna_output=[x.numpy().std(axis=2, ddof=0).reshape(-1)])
+
+  @unittest.skip
+  def test_reduceops_order(self):
```

Comment:
this tests if reduceops different ways the linearizer can be given a set of reduceops and makes sure they all end up in the correct order